### PR TITLE
Reinforced Plasma Glass is more SME-worthy

### PR DESCRIPTION
**Pull request by ghost**
Reinforced plasma windows need to be more SME worthy. Also max_integrity of rplasma windows is now twice of normal reinforced windows

[THIS MAY NEED TO BE TEST/SPEEDMERGED? DUE TO REINFORCED PLASMA WINDOWS CURRENTLY BEING USED BY DEFAULT IN THE SME SETUP]

🆑 More Robust Than You
tweak: Reinforced Plasma windows now have more integrity than reinforced windows
/🆑

### CODE REVIEW

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -420,6 +420,7 @@
 	heat_resistance = 1600
 	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 25, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 100)
 	max_integrity = 50
+	obj_integrity = 50
```
Comment (Cyberboss) [inline code/game/objects/structures/window.dm]: Don't set obj_integrity: #29030

File: code/game/objects/structures/window.dm [hunk 2]
```diff
@@ -433,7 +434,8 @@
 	reinf = FALSE
 	heat_resistance = 25000
 	armor = list("melee" = 75, "bullet" = 5, "laser" = 0, "energy" = 0, "bomb" = 45, "bio" = 100, "rad" = 100, "fire" = 00, "acid" = 100)
-	max_integrity = 50
+	max_integrity = 95
+	obj_integrity = 95
```
Comment (Cyberboss) [inline code/game/objects/structures/window.dm]: Don't set obj_integrity: #29030

File: code/game/objects/structures/window.dm [hunk 3]
```diff
@@ -447,7 +449,8 @@
 	reinf = TRUE
 	heat_resistance = 50000
 	armor = list("melee" = 85, "bullet" = 20, "laser" = 0, "energy" = 0, "bomb" = 60, "bio" = 100, "rad" = 100, "fire" = 99, "acid" = 100)
-	max_integrity = 100
+	max_integrity = 200
+	obj_integrity = 200
```
Comment (Cyberboss) [inline code/game/objects/structures/window.dm]: Don't set obj_integrity: #29030

File: code/game/objects/structures/window.dm [hunk 4]
```diff
@@ -469,6 +472,7 @@
 	icon_state = "window"
 	dir = FULLTILE_WINDOW_DIR
 	max_integrity = 50
+	obj_integrity = 50
```
Comment (Cyberboss) [inline code/game/objects/structures/window.dm]: Don't set obj_integrity: #29030

File: code/game/objects/structures/window.dm [hunk 5]
```diff
@@ -482,7 +486,8 @@
 	icon = 'icons/obj/smooth_structures/plasma_window.dmi'
 	icon_state = "plasmawindow"
 	dir = FULLTILE_WINDOW_DIR
-	max_integrity = 60
+	max_integrity = 100
+	obj_integrity = 100
```
Comment (Cyberboss) [inline code/game/objects/structures/window.dm]: Don't set obj_integrity: #29030

File: code/game/objects/structures/window.dm [hunk 6]
```diff
@@ -496,7 +501,8 @@
 	icon = 'icons/obj/smooth_structures/rplasma_window.dmi'
 	icon_state = "rplasmawindow"
 	dir = FULLTILE_WINDOW_DIR
-	max_integrity = 120
+	max_integrity = 500
+	obj_integrity = 500
```
Comment (Cyberboss) [inline code/game/objects/structures/window.dm]: Don't set obj_integrity: #29030

File: code/game/objects/structures/window.dm [hunk 7]
```diff
@@ -510,6 +516,7 @@
 	icon_state = "r_window"
 	dir = FULLTILE_WINDOW_DIR
 	max_integrity = 100
+	obj_integrity = 100
```
Comment (Cyberboss) [inline code/game/objects/structures/window.dm]: Don't set obj_integrity: #29030

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by MoreRobustThanYou** [comment]
Ok, done

**Review by Cyberboss — APPROVED** [review]

**Comment by MoreRobustThanYou** [comment]
I only did that because of the bug making windows near broken at the start

**Comment by ShizCalev** [comment]
Huh.
I was looking at that and thought the integrity values were pretty low. I figured the reasoning for it was because there are high armor values on them instead.

Just to note, high pressure windows had an integrity of 1000.

File: code/game/objects/structures/window.dm [hunk 8]
```diff
@@ -496,7 +496,7 @@
 	icon = 'icons/obj/smooth_structures/rplasma_window.dmi'
 	icon_state = "rplasmawindow"
 	dir = FULLTILE_WINDOW_DIR
-	max_integrity = 120
+	max_integrity = 500
```
Comment (Cyberboss) [inline code/game/objects/structures/window.dm]: Buff this to 1000 then

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**